docs/user-guide/kubectl-conventions.md

```diff
@@ -35,7 +35,8 @@ In order for `kubectl run` to satisfy infrastructure as code:
 
 * Pod - use `run-pod/v1`.
 * Replication controller - use `run/v1`.
-* Deployment - use `deployment/v1beta1`.
+* Deployment, using `extensions/v1beta1` endpoint - use `deployment/v1beta1`.
+* Deployment, using `apps/v1beta1` endpoint - use `deployment/apps.v1beta1`.
```

Comment:
Put `(default)` after `deployment/v1beta1` and `(recommended)` after `deployment/apps.v1beta1`, or just remove `deployment/v1beta1` since we want to deprecate it. 

@kargakis WDYT?